src/configure/helper/LocalGitRepoHelper.ts

```diff
@@ -23,6 +25,8 @@ export class LocalGitRepoHelper {
             return repoService;
         }
         catch(error) {
+            let gitFolderExists = fs.existsSync(path.join(repositoryPath, ".git"));
```

Comment:
why do we need to check this ? .git folder can be in a parent folder

Reply:
As per telemetry, lot of users are seeing error 'Not a git repo'. We want to see if in some cases the module we are using fails to recognize a git repo. Yes .git folder can be in parent folders as well, this is just to get confirmation that there are NO cases where user sees this error message but has a .git folder in root.